README.md

```diff
@@ -1 +1,31 @@
-# docker-vault-init
\ No newline at end of file
+# Vault Init
+
+[![Build Status](https://travis-ci.org/Intellection/vault-init.svg?branch=master)](https://travis-ci.org/Intellection/vault-init)
+
+This is a tool to automate initialising Vault, encrypting the resulting root token with AWS KMS, and storing that token on AWS S3.
+
+## Development
+
+The instructions below are only necessary if you intend to work on the source code.
+
+### Requirements
+
+1. Ensure that you have a [properly
+   configured](https://golang.org/doc/code.html#Workspaces) Go workspace.
+1. For dependency management, you'll need `dep` which can be installed with
+   `brew install dep`.
+
+### Building
+
+1. Fetch the code with `go get -v github.com/Intellection/vault-init`.
+1. Install the Go development tools via `make tools`.
+1. Install `dep` via `curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh`. `brew install dep` can also be used if using macOS.
+1. Install application dependencies via `make dependencies` (they'll be placed
+   in `./vendor`). Requires [golang/dep][dep] package manager.
+1. Build and install the binary with `make build tag=x.y.z`.
+1. Run the command e.g. `./bin/vault-init help` as a basic test.
```

Comment:
This should be removed since there's no `help` command ...

```console
$ ./bin/vault-init help
Initialising Vault...
panic: Put http://127.0.0.1:8200/v1/sys/init: dial tcp 127.0.0.1:8200: connect: connection refused

goroutine 1 [running]:
github.com/Intellection/vault-init/cmd.checkError(...)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:131
github.com/Intellection/vault-init/cmd.initVault(0x0, 0x0)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:164 +0x673
github.com/Intellection/vault-init/cmd.glob..func1(0x1d02f00, 0xc0000bb780, 0x1, 0x1)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:75 +0x9b
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).execute(0x1d02f00, 0xc0000cc010, 0x1, 0x1, 0x1d02f00, 0xc0000cc010)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:830 +0x2aa
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1d02f00, 0x103cb9a, 0x1cc8760, 0xc000000180)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/Intellection/vault-init/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/vendor/github.com/spf13/cobra/command.go:864
github.com/Intellection/vault-init/cmd.Execute()
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/cmd/root.go:119 +0x31
main.main()
        /Users/Kingori/Workspace/src/github.com/Intellection/vault-init/main.go:26 +0x20
```